### PR TITLE
Rewrite repo readme

### PR DESCRIPTION
**Pull request by Joehuu**
Fixed grammar, removed stuff that people don't know or should know of (i.e. the old wiki, the GitHub account, the PR branch name thing), removed repetitive things, and formatting.

Added a new intro commented out until the main wiki page gets implemented.

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 
-1. A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/ "GitHub Help in Signing up an account").
-2. A general understanding of the feature-branch workflow.
-3. Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help on Writing on GitHub").
-4. Ability to write in English, preferably British English.
+1. General understanding of the feature-branch workflow (i.e. pull request a branch that is not "master").
+2. Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help: Writing on GitHub").
+3. Ability to write in English, preferably British English.
 
 ## Getting Started
```
Comment (FreezyLemon) [inline README.md]: I think you should either change this to a "helpful links to get started" section and make the contents a list or, alternatively, change these sentences to be grammatically complete. As is, the header hints at a guide of sorts, but you just list some links instead.

Comment (Joehuu) [reply README.md]: I'll just make a sub header called `Helpful Links` and list them, since the title you mention is too long.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+
+Home of the [osu!](https://osu.ppy.sh) wiki.
 
 ## Requirements
 
-1. A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/ "GitHub Help in Signing up an account").
-2. A general understanding of the feature-branch workflow.
-3. Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help on Writing on GitHub").
-4. Ability to write in English, preferably British English.
+1. General understanding of the feature-branch workflow (i.e. pull request a branch that is not "master").
```
Comment (TPGPL) [inline README.md]: i.e. -> e.g.
make a pull request from a branch?
"master" -> `master`

File: README.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 ## Getting Started
 
-Contributions to the osu! wiki are now done through GitHub.
+### Helpful Links
 
-A basic guide to using GitHub, specifically tailored for the osu! wiki project, is available at the [osu! wiki Contribution Guide](https://osu.ppy.sh/help/wiki/osu!_wiki_contribution_guide).
+- The [osu! wiki Contribution Guide](https://osu.ppy.sh/help/wiki/osu!_wiki_Contribution_Guide) for help with using GitHub, specifically tailored for the osu! wiki project.
 
-The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available at the [Article Styling Criteria](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria).
+- The [Article Styling Criteria](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria), which *must* be adhered to for all new and rewritten pages going forward.
```
Comment (TPGPL) [inline README.md]: rather bold than italic `must`?
pages -> articles

**Pull request merged**